foundation-ui/services/AuthorizeBundle.js

```diff
@@ -0,0 +1,111 @@
+import React, {PropTypes} from 'react';
```

Comment:
Let's reorganize the file structure as described in https://github.com/dcos/dcos-ui/pull/1130#discussion_r80878513